### ISSUE TITLE
A Unit must be only once sub-Unit of a Unit

### ISSUE THREAD

**Issue by teamcsdc** [guaraqe/csdc#41]
we must look in the database what's already done before putting new values.
Actually, when you click on a Unit Unit-viewer, you can find more than one time the same unit if the submission has been done (by error or not) more than one time

**Comment by guaraqe** [comment]
This seems to be a good first backend task for one of you, since it is very self contained, touches only one component, and has a clear completion criterion.

Open a PR in this repo when ready.
